proto/commons/massa/model/v1/amount.proto

```diff
@@ -17,7 +19,7 @@ option swift_prefix = "MMODEL";
 // the future. value = mantissa / (10^scale)
 message NativeAmount {
   // Mantissa
-  fixed64 mantissa = 1;
+  google.protobuf.UInt64Value mandatory_mantissa = 1;
```

Comment:
Hello, 
I think that here, we don't want to make this fields optional(mantissa, mandatory_scale)

We use Wrapped types to replace: optional uint64(protobuf native types like string, uintXX, bool) , but for mandatory fields, we'll continue to use uint64

Reply:
I need optional here, even if the field is mandatory.
That's the only way I can be sure that 0 does not mean None